ci-operator/config/coreos/rhel-coreos-config/coreos-rhel-coreos-config-rhel-9.6.yaml

```diff
@@ -0,0 +1,132 @@
+      memory: 200Mi
+  rhcos-9-build-test-metal:
+    limits:
+      devices.kubevirt.io/kvm: "1"
+    requests:
+      cpu: 2000m
+      devices.kubevirt.io/kvm: "1"
+      ephemeral-storage: 40Gi
+      memory: 3Gi
+  rhcos-9-build-test-qemu:
+    limits:
+      devices.kubevirt.io/kvm: "1"
+    requests:
+      cpu: 2000m
+      devices.kubevirt.io/kvm: "1"
+      ephemeral-storage: 40Gi
+      memory: 3Gi
+  rhcos-9next-build-test-metal:
+    limits:
+      devices.kubevirt.io/kvm: "1"
+    requests:
+      cpu: 2000m
+      devices.kubevirt.io/kvm: "1"
+      ephemeral-storage: 40Gi
+      memory: 3Gi
+  rhcos-9next-build-test-qemu:
+    limits:
+      devices.kubevirt.io/kvm: "1"
+    requests:
+      cpu: 2000m
+      devices.kubevirt.io/kvm: "1"
+      ephemeral-storage: 40Gi
+      memory: 3Gi
+  scos-9-build-test-metal:
+    limits:
+      devices.kubevirt.io/kvm: "1"
+    requests:
+      cpu: 2000m
+      devices.kubevirt.io/kvm: "1"
+      ephemeral-storage: 40Gi
+      memory: 3Gi
+  scos-9-build-test-qemu:
+    limits:
+      devices.kubevirt.io/kvm: "1"
+    requests:
+      cpu: 2000m
+      devices.kubevirt.io/kvm: "1"
+      ephemeral-storage: 40Gi
+      memory: 3Gi
+tests:
+- as: validate
+  capabilities:
+  - kvm
+  commands: ci/prow-entrypoint.sh validate
+  container:
+    from: src
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+- as: rhcos-9-build-test-qemu
+  capabilities:
+  - kvm
+  commands: /src/ci/prow-entrypoint.sh rhcos-9-build-test-qemu
+  container:
+    from: build-image
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+- as: rhcos-9-build-test-metal
+  capabilities:
+  - kvm
+  commands: /src/ci/prow-entrypoint.sh rhcos-9-build-test-metal
+  container:
+    from: build-image
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+- as: rhcos-9next-build-test-qemu
+  capabilities:
+  - kvm
+  commands: /src/ci/prow-entrypoint.sh rhcos-9next-build-test-qemu
+  container:
+    from: build-image
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+- as: rhcos-9next-build-test-metal
+  capabilities:
+  - kvm
+  commands: /src/ci/prow-entrypoint.sh rhcos-9next-build-test-metal
+  container:
+    from: build-image
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+- as: scos-9-build-test-qemu
+  capabilities:
+  - kvm
+  commands: /src/ci/prow-entrypoint.sh scos-9-build-test-qemu
+  container:
+    from: build-image
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+- as: scos-9-build-test-metal
+  capabilities:
+  - kvm
+  commands: /src/ci/prow-entrypoint.sh scos-9-build-test-metal
+  container:
+    from: build-image
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+zz_generated_metadata:
+  branch: rhel-9.6
+  org: coreos
+  repo: rhel-coreos-config
```

Comment:
I think we should delete all of this too.

We're really only focused on testing one stream on this branch now and it's `rhel-9.6`.